tests/integration/dual_test.json

```diff
@@ -7,4 +7,4 @@
     "db_settings": "{\"message_table\": \"message_int_test\",\"router_table\": \"router_int_test\"}",
     "dsn": "http://localhost:8000/"
   }
-}
+}
```

Comment:
Was removing this last \n intentional?

Reply:
My auto format for some reason did so! Probably an IDE "feature". Is there a preference one way or another? Happy to change it if we prefer a trailing line.

Reply:
no worries. It's a JSON file so not super critical. Just curious if there was some different reason.